synapseclient/table.py

```diff
@@ -622,10 +622,6 @@ def _csv_to_pandas_df(
     test_import_pandas()
     import pandas as pd
 
-    # DATEs are stored in csv as unix timestamp in milliseconds
-    def datetime_millisecond_parser(milliseconds):
-        return pd.to_datetime(milliseconds, unit="ms", utc=True)
-
     if not date_columns:
```

Comment:
This if statement can probably be removed.

Reply:
@linglp see this comment. I don't think you need to set the date_columns to an empty list any longer.

Reply:
Thanks Tom! I have deleted the if not statement.